conans/client/tools/system_pm.py

```diff
@@ -83,7 +85,7 @@ def update(self):
         """
             Get the system package tool update command
         """
-        mode = self._get_sysrequire_mode()
+        mode = self._get_sysrequire_mode(self._default_mode)
```

Comment:
This is in general an anti-pattern, you shouldn't pass to a self.method a self.variable